app/src/pages/auth/LoginPage.tsx

```diff
@@ -20,12 +20,12 @@ export const LoginPage: React.FC<any> = () => {
     console.log("Rendering page");
     return (
         <IonPage>
-            <IonContent>
-                <div className="bg-gray-900 h-full w-full items-center flex justify-center flex-col">
+            <IonContent color="new">
+                <div className=" h-full w-full items-center flex justify-center flex-col">
                     <div className="text-center my-4">
-                        <h1 className="text-4xl text-gray-100 font-bold ">Industry 4.0</h1>
-                        <h2 className="text-2xl text-gray-300">
-                            <em>Monitor your machines like never before.</em>
+                        <IonTitle className="text-4xl text-gray-100 font-heading">Industry 4.0</IonTitle>
```

Comment:
It'd be good to have the same font on this page as we had in the UI design `https://fonts.googleapis.com/css?family=Share Tech Mono`. I've added that back into the heading component as well in pr #56